index.html

```diff
@@ -5,36 +5,35 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="author" content="Austin">
         <link rel="stylesheet" type="text/css" href="styles.css"> 
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins"/>
+
     </head>
     <body>
       <header>
-        <nav>
+          <nav>
             <div class="logo-container">
                 <a href="#">
                     <h3 id="logo">My Logo</h3>
                 </a>
             </div>
-            <ul class="nav-links">
-                <li class="navlinks">Portfolio</li>
-                <li class="navlinks">About</li>
-                <li class="navlinks">Contact</li>
-                <li class="navlinks"><img src="img/Union.png" alt="Union"/></li>
-            </ul>
-        </nav>
+               
+            <label for="check" class="hamberger-btn">
+             <img id="ham" href="#" src="img/IconMenu.png">
+            </label>
+          </nav>
       </header>
 
       <section class="section">
         <div class="container">
-           <div class="headline">
-            <h3 id="myName">
```

Comment:
Please consider adding a proper `background image` to the section as can see in the design template to make it look alike to the design.

### YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235967291-3334aaa2-5794-4209-aa34-a06e05eb02e9.png)


### FIGMA DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235967301-87c1f5f0-e38b-4139-96b5-0aa6556a6b6c.png)